config/ext_jenkins-infra.yaml

```diff
@@ -491,6 +491,17 @@ controller:
                       credentialsId('github-app-infra')
 
                       traits {
+                        gitHubSCMSourceStatusChecksTrait {
+                            // Note: changing this name might have impact on github branch protections if they specify status names
+                            name('[infra.ci.jenkins.io] ' + config.name)
```

Comment:
do we need this? what does it look like rendered?

Reply:
Looks like this:

<img width="956" alt="Capture d’écran 2022-03-17 à 18 58 44" src="https://user-images.githubusercontent.com/1522731/158866339-ee90c5e4-ef28-4ef6-9cbb-01a27839b3ad.png">

We found it pretty useful to immediately see which controller is used.